cucumber/authenticators_gcp/features/authn_gcp_token_errors.feature

```diff
@@ -69,21 +69,6 @@ Feature: GCP Authenticator - Test Token Error Handling
     CONJ00009E Field 'jwt' is missing or empty in request body
     """
 
-  # "authn-gcp/project-id" annotation is set because at least one of the annotations is expected.
-  # we do not test with other annotations as the test will fail on the token validation
-  Scenario: Authenticate using token in standard format is denied
```

Comment:
@nessiLahav what is the expected behaviour in this use-case? do we want to verify it?

Reply:
Yes sure this is part of the test plan, Elad should review it